features/distro-resources/src/main/resources/items/demo.items

```diff
@@ -1,23 +1,23 @@
-Group gFF           "First Floor"   <firstfloor>
-Group gGF           "Ground Floor"  <groundfloor>
-Group gC            "Cellar"        <cellar>
-Group Garden        "Garden"        <garden>
+Group gFF           "First Floor"   <firstfloor>    ["FirstFloor"]
+Group gGF           "Ground Floor"  <groundfloor>   ["GroundFloor"]
+Group gC            "Cellar"        <cellar>        ["Cellar"]
+Group Garden        "Garden"        <garden>        ["Garden"]
 Group Weather       "Weather"       <sun>
 
 Group Status
 Group Shutters
 
-Group GF_Living     "Living Room"   <video>     (gGF)
-Group GF_Kitchen    "Kitchen"       <kitchen>   (gGF)
-Group GF_Toilet     "Toilet"        <bath>      (gGF)
-Group GF_Corridor   "Corridor"      <corridor>  (gGF)
+Group GF_Living     "Living Room"   <video>     (gGF)   ["LivingRoom"]
+Group GF_Kitchen    "Kitchen"       <kitchen>   (gGF)   ["Kitchen"]
+Group GF_Toilet     "Toilet"        <bath>      (gGF)   ["Room"] { synonyms="Toilets,WC,Restroom" }
```

Comment:
I'd suggest to mark this also as "Bathroom" - in English, that is the general term (e.g. guest will ask you for your bathroom if they need a toilet).

Reply:
I just gave it a try, thinking you were referring to synonyms - this doesn't work since the SemanticsService will resolve "bathroom" to the "Bathroom" tag so it won't consider the synonyms